bootstrap/runner.py

```diff
@@ -275,8 +275,8 @@ def format_files(self):
             [
                 "python3",
                 "-m",
-                "black",
-                "-q",
+                "ruff",
+                "format",
```

Comment:
We said we'd leave out the boostraps from upgrading to `ruff format` because of that bug in `ruff` itself.

Reply:
It seems to be a misconfiguration of our requirements rather than a bug in Ruff itself.

Reply:
`ruff` here is used to format the newly generated project code and not for the repo code.

Reply:
> It seems to be a misconfiguration of our requirements rather than a bug in `ruff` itself. It was necessary to move `ruff` into the `commons.in`

It's a confirmed bug by Astral.